tools/db_bench_tool.cc

```diff
@@ -6455,9 +6455,7 @@ class Benchmark {
 
       // Pick a Iterator to use
       size_t db_idx_to_use =
-          (db_.db == nullptr)
-              ? (size_t{thread->rand.Next()} % multi_dbs_.size())
-              : 0;
+          (db_.db == nullptr) ? thread->rand.Next() % multi_dbs_.size() : 0;
```

Comment:
Why this change?  It is not like the others....

Reply:
It didn't compile on 32-bit (warning as error)